src/pages/iou/IOUModal.js

```diff
@@ -138,6 +179,33 @@ class IOUModal extends Component {
         this.setState({selectedCurrency});
     }
 
+    closeModal() {
+        Navigation.dismissModal();
+    }
+
+    createTransaction({debtorEmail, splits, participants}) {
```

Comment:
Can't we get `debtorEmail` and `participants` from the IOUModal object data? I don't think we need to pass it back from IOUConfirm